src/App.jsx

```diff
@@ -65,100 +58,127 @@ const styles = theme => ({
+
+      curr.group = key;
+      curr.projects = Object.entries(value).reduce((projects, project) => {
+        const [key, proj] = project;
+
+        proj.key = key;
+        projects.push(proj);
+        projectSelected[key] = false;
+
+        return projects;
+      }, []);
+      allProjectGroupsEntries.push(curr);
+    });
+
     this.state = {
       mobileOpen: false,
-      allProjects : [...data.Projs],
+      allProjectGroupsEntries,
+      projectSelected,
     };
   }
 
-
   handleDrawerToggle = () => {
     this.setState({ mobileOpen: !this.state.mobileOpen });
   };
 
-  handleCheckboxToggle = (proj) => {
-    var allProjects = [...this.state.allProjects];
+  handleCheckboxToggle = key => {
+    const { projectSelected } = this.state;
 
-    allProjects.forEach(function(project, index) {
-      if (project[0] === proj) {
-        console.log(project[2])
-        project[2] = !project[2];
-      }
-    });
-
-    this.setState({allProjects: allProjects});
+    projectSelected[key] = !projectSelected[key];
+    this.setState({ projectSelected });
   };
 
   render() {
     const { classes, theme } = this.props;
-    const { projectSelected, allProjects } = this.state;
-
+    const { allProjectGroupsEntries, projectSelected } = this.state;
     const drawer = (
       <div>
         <div className={classes.toolbar} />
-          <FormControl component="fieldset">
-            <FormLabel component="legend">Are you interested in:</FormLabel>
-            <FormGroup>
-              { allProjects.map((proj, index) => (              
-                <FormControlLabel
-                  control={
-                    <Checkbox
-                      value={proj[0]}
-                      checked = {proj[2]}
-                      onChange = {(e) => this.handleCheckboxToggle(proj[0],e)}
-                    />
-                  }
-                  key={index}
-                  label={proj[0]}
-                />
-              ))}
-            </FormGroup>
-
-            <FormLabel component="legend">Do you know ?</FormLabel>
-            <FormGroup>
-              { data.Languages.map((lang, index) => (              
-                <FormControlLabel
-                  control={
-                    <Checkbox
-                      value={lang}
-                    />
-                  }
-                  key={index}
-                  label={lang}
-                />
-              ))}
-            </FormGroup>
-
-            <FormLabel component="legend">Filter result on:</FormLabel>
-            <FormGroup>
-              { bugType.map((bug, index) => (
-                <FormControlLabel
-                  control={
-                    <Checkbox
-                      value={bug}
-                    />
-                  }
-                  label={bug}
-                  key={index}
-                />
+        <FormControl component="fieldset">
+          <FormLabel component="legend">Are you interested in:</FormLabel>
```

Comment:
I think `FormControl` is usually used for form inputs. The drawer can be done using plain `div`s. This is how we've done it in tc-web https://github.com/taskcluster/taskcluster-web/blob/4e75cb216cdede3fc40e75837e6b7b94f180c761/src/components/Dashboard/index.jsx#L163-L187.